src/hooks/useSnackbar.ts

```
@@ -0,0 +1,88 @@
+
+    /**
+     * Displays a snackbar notification with the given message and options.
+     * @param message The message to display in the snackbar.
+     * @param severity The severity of the message. Can be "info", "success", "warning", or "error".
+     * @param autoHideDuration The duration in milliseconds for which the snackbar should stay open.
+     *     If not provided, defaults to 6000ms.
+     */
+    const showMessage = (
+        message: string,
+        severity: SnackbarSeverity = 'info',
+        autoHideDuration: number = 6000,
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)